capa/ghidra/capa_explorer.py

```diff
@@ -137,84 +137,114 @@ def set_pre_comment(self, ghidra_addr, sub_type, description):
         else:
             return
 
-    def label_matches(self):
+    def label_matches(self, namespace=False, comments=False):
```

Comment:
Let's split this into two functions, `create_capa_ namespace` and `create_capa_comments`, that are only called if the corresponding bool values retrieved from the user are `True`.